src/test/shared/codelens/localLambdaRunner.test.ts

```diff
@@ -378,4 +385,51 @@ describe('localLambdaRunner', async () => {
             })
         }
     })
+
+    describe('makeBuildDir', () => {
+        it ('creates a temp directory', async () => {
+            const dir = await localLambdaRunner.makeBuildDir()
+            assert.strictEqual(fsUtils.fileExists(dir), true)
```

Comment:
assert that `dir` is not empty

Reply:
The directory should be empty.

Reply:
✔️ to the above

Reply:
What I meant was "add an assert that the variable `dir` is not an empty string"

Reply:
Bah, gotcha

Reply:
✔️